models/schemas/Coding-CourseOfTherapyType.yaml

```diff
@@ -8,8 +8,8 @@ properties:
     type: string
     description: The identification of the code system that defines the meaning of the symbol in the code.
     enum: 
-      - "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType"
-    example: "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType"
+      - "https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy"
```

Comment:
The enum will need to allow for both code systems:
- http://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy
- https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy

Reply:
Added both to the spec